docs/getting-started/first-trigger.md

```diff
@@ -21,18 +24,28 @@
           kind: Service
           name: cloudevents-player
     ```
-==**Expected Output**==
-```{ .bash .no-copy }
-Trigger 'cloudevents-player' successfully created in namespace 'default'.
-```
 
-??? question "What CloudEvents is my `Trigger` listening for?"
-    Since we didn't specify a `--filter` in our `kn` command, our Trigger is listening for any CloudEvents coming into the `Broker`.
+    Once you've created your YAML file (named something like "ce-trigger.yaml"):
+    ``` bash
+    kubectl apply -f ce-trigger.yaml
+    ```
+
+    ==**Expected Output**==
+    ```{ .bash .no-copy }
+    trigger.eventing.knative.dev/cloudevents-trigger created
+    ```
+
+
+trigger.eventing.knative.dev/cloudevents-player created
+??? question "What CloudEvents is my Trigger listening for?"
+    Since we didn't specify a `--filter` in our `kn` command, our Trigger is listening for any CloudEvents coming into the Broker.
```

Comment:
Because we didn't specify a `--filter` in our `kn` command, the Trigger is listening for any CloudEvents coming into the Broker.